config/locales/en/course/assessment/assessments.yml

```diff
@@ -56,6 +56,7 @@ en:
         form:
           autograded: 'Automatically assign grade and experience points after finalization'
           autograded_hint: 'Note that all the questions in the assessment must be auto-gradable'
+          skippable: 'Allow students to skip steps'
```

Comment:
Maybe leave it here first ? It's the same text as v1, so at least v1 users won't be confused.

I am also working on a tutorial UI for creating the assessment, that would help with this.